extensions/desktop/kde-neon-6/launcher-specific

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
for the gnome extension we went with putting these scripts in the content snap itself, to make changes and updates easier. I understand that can be a lot more work, so if it is too much effort now, I would at least encourage it for the core24 version of these snaps.

Not going to block on this

Reply:
Will do for core24, thanks